packages/client/tests/functional/_utils/setupFilesAfterEnv.ts

```diff
@@ -1,8 +1,11 @@
+import { faker } from '@faker-js/faker'
 import { toMatchInlineSnapshot, toMatchSnapshot } from 'jest-snapshot'
 import stripAnsi from 'strip-ansi'
 
 process.env.PRISMA_HIDE_PREVIEW_FLAG_WARNINGS = 'true'
 
+faker.seed([process.pid, Date.now()])
```

Comment:
I'm curious, what's this for?

Reply:
I had a hypothesis that different process have different jest workers might accidentally get synchornized random generators (faker uses it's own implementation rather that `Math.random`) and that's why we occasionally get unique constraint violations for the test that creates a single record. So, this was an attempt to explicitly make process id a part of the seed of the PRNG. I don't think that was the real issue though, judging by the last run, so it will probably be reverted.